api/provider/aws/service_read_test.go

```diff
@@ -0,0 +1,90 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/api/tag"
+	"github.com/quintilesims/layer0/common/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestService_lookupDeployIDFromTaskDefinition(t *testing.T) {}
```

Comment:
WIP?

Reply:
Oh crap! Yeah, amidst all our debating, I completely forgot that I hadn't finished this yet.

Reply:
✅